gnosis/eth/ethereum_network.py

```diff
@@ -204,6 +204,122 @@ class EthereumNetwork(Enum):
     SHIDEN = 336
     CRONOS_MAINNET = 25
     CRONOS_TESTNET = 338
+    UBIQ_NETWORK_TESTNET = 2
```

Comment:
Ubiq testnet has id 9. https://chainlist.org/chain/2

Reply:
Thanks! It's 9 now.